### DISCUSSION TITLE
How to use the  'Touch Repeat Directive'

### DISCUSSION THREAD

**Discussion by RelievedCy** [zauberzeug/nicegui#3889] [Q&A]
### Description

~~~python
from nicegui import ui
with ui.card() as card:
	ui.label("touch me").classes("border-2 custom-area cursor-pointer bg-primary text-white shadow-2 relative-position "
	                             "row flex-center").props(
	
	).on(
		"v-touch-hold.mouse", lambda: ui.notify("1")
	)
card._props['v-touch-hold.mouse'] = '()=> console.log("touch")'
ui.run(

)
~~~
This method currently has no effect, how can it be used？
[quasar:touch-repeat](https://quasar.dev/vue-directives/touch-repeat)
![image](https://github.com/user-attachments/assets/f63d493c-5141-46f6-9ced-7c73a9380625)

**Comment by falkoschindler** [comment]
Good question, @RelievedCy!

We probably need to define it as a dynamic prop:
```py
ui.button('touch me', on_click=lambda: print('click')) \
    .props(''' :v-touch-repeat.mouse="() => { console.log('touch') }" ''')
```
But so far it doesn't work. Does anyone else have an idea?

**Comment by RelievedCy** [comment]
Process prop in nicegui as string？
